lib/krane/cluster_resource_discovery.rb

```diff
@@ -26,7 +26,7 @@ def prunable_resources(namespaced:)
         group_versions = api_versions[resource['apigroup'].to_s]
         version = version_for_kind(group_versions, resource['kind'])
         [resource['apigroup'], version, resource['kind']].compact.join("/")
-      end.compact
+      end.compact.uniq { |gvk| gvk.split("/").last }
```

Comment:
This ends up being an issue because ` k api-resources` returns a kind multiple times if it appears in multiple groups. I think the real fix is `fetch_resources(namespaced: namespaced).uniq { |r| r['kind']}.map`